src/tools/miri/tests/pass/issues/issue-miri-2068.rs

```diff
@@ -1,5 +1,3 @@
-#![feature(pin_macro)]
-
```

Comment:
Hmm, I don't know what's the procedure with a sub-tree; should I have made a change to `miri` beforehand?

Reply:
Nope, this is exactly how we expect these changes to happen. No more need to sync things separately, just do everything in one PR